bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/panel/BootstrapGenericPanel.html

```diff
@@ -0,0 +1,27 @@
+<html xmlns:wicket="http://wicket.apache.org">
+    <head>
+        <title>BootstrapGenericPanel</title>
+    </head>
+    <body>
+        <wicket:panel>
+        	
+        	<wicket:enclosure child="panelTitle">
+        		<div class="panel-heading">
+        			<h3 class="panel-title"><span wicket:id="panelTitle"></span></h3>
+        		</div>
+        	</wicket:enclosure>
+        	
+			
+			<wicket:enclosure child="panelBody">
```

Comment:
There is no need of `<wicket:enclosure>` here.
As a general rule: try to avoid using enclosures because they have many known issues (consult with Wicket JIRA about them).